x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/configure_pipeline.tsx

```diff
@@ -27,6 +27,11 @@ import { docLinks } from '../../../../../shared/doc_links';
 
 import { MLInferenceLogic } from './ml_inference_logic';
 
+const NO_SOURCE_FIELDS_ERROR = i18n.translate(
+  'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.configure.sourceField.error',
+  { defaultMessage: 'Select a text or keyword field from your index field mapping' }
```

Comment:
This copy could probably be better?

Reply:
I'd like to see something that explicitly says there is no schema/field mapping and the user needs to add that for fields to show up there.

Reply:
Agree - if only to be shown in the error condition, perhaps something like: "The target index does not contain a field mapping", and maybe linking to the [mapping documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping.html)?

Reply:
Updated to:

```
This index contains an empty field mapping. <a>Learn more about field mapping</a>
```

Reply:
I feel like it could use another sentence in the middle to give a bit more context on the error.

"You need select a field to configure the pipeline."

Reply:
I like that @brianmcgue

Reply:
LOVE IT

Reply:
updated.